docker-compose/docker-compose-injistack/certify_init.sql

```diff
@@ -0,0 +1,166 @@
+                                    upd_dtimes timestamp,
+                                    CONSTRAINT pk_svgtmp_id PRIMARY KEY (id)
+);
+
+CREATE TABLE certify.template_data(
+                                    context character varying(1024) NOT NULL,
+                                    credential_type character varying(512) NOT NULL,
+                                    template VARCHAR NOT NULL,
+                                    cr_dtimes timestamp NOT NULL default now(),
+                                    upd_dtimes timestamp,
+                                    CONSTRAINT pk_template PRIMARY KEY (context, credential_type)
+);
+
+INSERT INTO certify.template_data (context, credential_type, template, cr_dtimes, upd_dtimes) VALUES ('https://vharsh.github.io/DID/mock-context.json,https://www.w3.org/2018/credentials/v1', 'MockVerifiableCredential,VerifiableCredential', '{
+    "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://vharsh.github.io/DID/mock-context.json"],
+    "issuer": "${issuer}",
+    "type": ["VerifiableCredential", "MockVerifiableCredential"],
+    "issuanceDate": "${validFrom}",
+    "expirationDate": "${validUntil}",
+    "credentialSubject": {
+        "gender": ${gender},
+        "postalCode": ${postalCode},
+        "fullName": ${fullName},
+        "dateOfBirth": "${dateOfBirth}",
+        "province": ${province},
+        "phone": "${phone}",
+        "addressLine1": ${addressLine1},
+        "region": ${region},
+        "vcVer": "${vcVer}",
+        "UIN": ${UIN},
+        "email": "${email}",
+        "face": "${face}"
+    }
+}', '2024-10-22 17:08:17.826851', NULL);
+INSERT INTO certify.template_data (context, credential_type, template, cr_dtimes, upd_dtimes) VALUES ('https://vharsh.github.io/DID/mock-context.json,https://www.w3.org/ns/credentials/v2', 'MockVerifiableCredential,VerifiableCredential', '{
+    "@context": [
+            "https://www.w3.org/ns/credentials/v2", "https://vharsh.github.io/DID/mock-context.json"],
+    "issuer": "${issuer}",
+    "type": ["VerifiableCredential", "MockVerifiableCredential"],
+    "validFrom": "${validFrom}",
+    "validUntil": "${validUntil}",
+    "credentialSubject": {
+    "gender": ${gender},
+        "postalCode": ${postalCode},
+        "fullName": ${fullName},
+        "dateOfBirth": "${dateOfBirth}",
+        "province": ${province},
+        "phone": "${phone}",
+        "addressLine1": ${addressLine1},
+        "region": ${region},
+        "vcVer": "${vcVer}",
+        "UIN": ${UIN},
+        "email": "${email}",
+        "face": "${face}"
+    }
+}', '2024-10-22 17:08:17.826851', NULL);
+INSERT INTO certify.template_data (context, credential_type, template, cr_dtimes, upd_dtimes) VALUES ('https://www.w3.org/2018/credentials/v1', 'FarmerCredential,VerifiableCredential', '{
+     "@context": [
+                "https://www.w3.org/2018/credentials/v1",
+                "https://vharsh.github.io/DID/farmer.json",
```

Comment:
can we move context to mosip repo instead of using personal repos?
I am referring to this https://vharsh.github.io/DID/farrmer.json

Reply:
Anyone changing the Usecase will have to update this to point to their repository